### ISSUE TITLE
CannotConnect sometimes lacks username

### ISSUE THREAD

**Issue by jpdillingham** [jpdillingham/Soulseek.NET#464]
```
Error handling server message: CannotConnect; Failed to read an integer (4 bytes) from position 4 of the message
```

Check to see if there's more data in the message before attempting to read the username string.

**Comment by ned-kelly** [comment]
Any chance we can get this one added into the docker build? - Perhaps have the docker hub builds built/pushed (latest) based on the master branch with your CircleCI - Then just version tag each release?

Keen to start playing with your API and seeing what magic I can get out of it, but this issue appears to still be in the latest build up on docker hub.

Cheers,

**Comment by jpdillingham** [comment]
Thanks for the reminder, I just pushed a new build and this problem should be resolved.  It wasn't affecting functionality, just a nuisance.  

I'm in the process of moving everything to a new repo here: https://github.com/slskd/slskd

The Docker images from that repo can be found here: https://hub.docker.com/r/slskd/slskd, and (I've lost track of what exactly) a few minor bugs have been fixed.

I'll eventually be deprecating the example in this repo once everything is set up.
